packages/help-center/src/styles.scss

```diff
@@ -6,14 +6,34 @@
 $head-foot-height: 50px;
 $z-index: 99999;
 
+@layer {
+	.help-center__container {
+		transition:
+			opacity 0.5s ease-in,
+			scale 0.5s ease-in,
+			display 1.5s ease-in;
+		/* stylelint-disable-next-line property-no-unknown */
+		transition-behavior: allow-discrete;
+
+		/* stylelint-disable-next-line scss/at-rule-no-unknown */
+		@starting-style {
+			opacity: 0;
+			scale: 1.1;
+		}
+
+		&[hidden] {
+			opacity: 0;
```

Comment:
Should probably use `visibilty` not `opacity` because this way the Help Center will still be clickable.

Reply:
This is for the animation effect. In the end its fading to display none. Or at least that is how I understand it. It will only be for .5 seconds before its removed from the DOM.